vector/src/main/res/menu/menu_timeline.xml

```diff
@@ -19,32 +19,36 @@
         android:title="@string/action_invite"
         app:showAsAction="never" />
 
+    <!-- We always want to show this item as an icon -->
     <item
         android:id="@+id/video_call"
         android:icon="@drawable/ic_video"
         android:title="@string/action_video_call"
         android:visible="false"
         app:iconTint="?colorPrimary"
-        app:showAsAction="ifRoom"
+        app:showAsAction="always"
         tools:visible="true" />
 
+    <!-- We always want to show this item as an icon -->
     <item
         android:id="@+id/voice_call"
         android:icon="@drawable/ic_phone"
         android:title="@string/call"
         android:visible="false"
         app:iconTint="?colorPrimary"
-        app:showAsAction="ifRoom"
+        app:showAsAction="always"
         tools:visible="true" />
 
+    <!-- We always want to show this item as an icon -->
     <item
         android:id="@+id/menu_timeline_thread_list"
         android:title="@string/action_view_threads"
         android:visible="false"
         app:actionLayout="@layout/view_thread_notification_badge"
         app:iconTint="?colorPrimary"
         app:showAsAction="always"
-        tools:visible="true" />
+        tools:visible="true"
+        tools:ignore="AlwaysShowAction" />
```

Comment:
Strangely adding this attribute in this `item` XML tag also has an effect on the warnings of the other `item` tags 🤔 .
Anyway for clarity, you should probably move this `tools:ignore` to the root XML tag `menu`.